stock_picking_backorder_strategy/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Picking backordering strategies',
+    'version': '12.0.1.0.1',
+    'development_status': 'Production/Stable',
```

Comment:
Remove this manifest key.

Reply:
Forget this comment, I just noticed these keys were incorporated to OCA's guidelines (https://github.com/OCA/maintainer-tools/commit/0a02dcc4).